R/fotmob_leagues.R

```diff
@@ -241,21 +239,18 @@ fotmob_get_league_matches <- function(country, league_name, league_id, cached =
   )
 }
 
-.fotmob_extract_data_from_page_props <- function(resp) {
```

Comment:
it seems that this no longer works... it's ok tho, since i only ever used this as a "backup" for extracting data